api/db/migrations/20250917000000_add-duration-column-to-complementary-certifications.js

```diff
@@ -0,0 +1,24 @@
+const TABLE_NAME = 'complementary-certifications';
+const COLUMN_NAME = 'duration';
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+const up = async function (knex) {
+  await knex.schema.table(TABLE_NAME, function (table) {
+    table.integer(COLUMN_NAME).nullable();
```

Comment:
je ne vois pas de cas fonctionnel ou on souhaiterait / pourrait avoir une valeur a  null. J'aime bien l'approche de toujours eviter autant que possible les null check

```suggestion
    table.integer(COLUMN_NAME).notNullable().default(0;
```